MSB6006 on Cuda 9.1 xmr stak Windows Compile

Hi there,

I got following error when trying compile xmr stak (lastest git).
`C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe" wurde mit dem Code 1 beendet. [C:\xmr\build\xmrstak_cuda_backend.vcxproj]`

# Basic information
  - Intel i7 - 5960X
  - NVidia 1080Ti

# Compile issues
  - Windows 10 Pro 1709

Here is the cmake output:

C:\xmr\build>cmake -G "Visual Studio 15 2017 Win64" -T v141,host=x64 ..
-- The C compiler identification is MSVC 19.12.25830.2
-- The CXX compiler identification is MSVC 19.12.25830.2
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx64/x64/cl.exe
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx64/x64/cl.exe -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx64/x64/cl.exe
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx64/x64/cl.exe -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Set miner currency to 'monero' and 'aeon'
-- Found CUDA: C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.1 (found suitable version "9.1", minimum required is "7.5")
-- Looking for CL_VERSION_2_0
-- Looking for CL_VERSION_2_0 - found
-- Found OpenCL: C:/Program Files (x86)/AMD APP SDK/3.0/lib/x86_64/OpenCL.lib (found version "2.0")
-- Found OpenSSL: C:/xmr-stak-dep/openssl/lib/libeay32.lib (found version "1.0.2m")
-- Configuring done
-- Generating done
-- Build files have been written to: C:/xmr/build

And this is also the the error in the end:
"C:\xmr\build\install.vcxproj" (Standardziel) (1) ->
"C:\xmr\build\ALL_BUILD.vcxproj" (Standardziel) (3) ->
"C:\xmr\build\xmrstak_cuda_backend.vcxproj" (Standardziel) (7) ->
(CustomBuild Ziel) ->
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe" wurde
mit dem Code 1 beendet. [C:\xmr\build\xmrstak_cuda_backend.vcxproj]

    40 Warnung(en)
    1 Fehler

I receive the same error when compiling on windows 10 1703  Intel i7 7700K Nvidia GTX 1070

Once it completes I can use the CPU miner but the nvidia.txt is not created and there is no configuration created. If I manually create the configuration it crashes during startup. This is with the last pull from the GIT repo as of 12/12/2017 

C:\xmr-stak\xmr-stak\build2>cmake -G "Visual Studio 15 2017 Win64" -T v141,host=x64 -DCPU_ENABLE=ON -DCUDA_ENABLE=ON ..
-- Selecting Windows SDK version 10.0.16299.0 to target Windows 10.0.15063.
-- The C compiler identification is MSVC 19.12.25830.2
-- The CXX compiler identification is MSVC 19.12.25830.2
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx64/x64/cl.exe
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx64/x64/cl.exe -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx64/x64/cl.exe
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.12.25827/bin/Hostx64/x64/cl.exe -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Set miner currency to 'monero' and 'aeon'
-- Found CUDA: C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.1 (found suitable version "9.1", minimum required is "7.5")
-- Looking for CL_VERSION_2_2
-- Looking for CL_VERSION_2_2 - not found
-- Looking for CL_VERSION_2_1
-- Looking for CL_VERSION_2_1 - not found
-- Looking for CL_VERSION_2_0
-- Looking for CL_VERSION_2_0 - not found
-- Looking for CL_VERSION_1_2
-- Looking for CL_VERSION_1_2 - found
-- Found OpenCL: C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.1/lib/x64/OpenCL.lib (found version "1.2")
-- Found OpenSSL: C:/xmr-stak-dep/openssl/lib/libeay32.lib (found version "1.0.2m")
-- Configuring done
-- Generating done
-- Build files have been written to: C:/xmr-stak/xmr-stak/build2

C:\xmr-stak\xmr-stak\build2>cmake -G "Visual Studio 15 2017 Win64" -T v141,host=x64 -DCPU_ENABLE=ON -DCUDA_ENABLE=ON -DOpenCL_ENABLE=ON ..
-- Selecting Windows SDK version 10.0.16299.0 to target Windows 10.0.15063.
-- Set miner currency to 'monero' and 'aeon'
-- Configuring done
-- Generating done
-- Build files have been written to: C:/xmr-stak/xmr-stak/build2

C:\xmr-stak\xmr-stak\build2>cmake --build . --config Release --target install
Microsoft (R) Build Engine version 15.5.180.51428 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 12/12/2017 6:59:05 PM.
Project "C:\xmr-stak\xmr-stak\build2\install.vcxproj" on node 1 (default targets).
Project "C:\xmr-stak\xmr-stak\build2\install.vcxproj" (1) is building "C:\xmr-stak\xmr-stak\build2\ZERO_CHECK.vcxproj" (2) on node 1 (default
targets).
PrepareForBuild:
  Creating directory "x64\Release\ZERO_CHECK\".
  Creating directory "x64\Release\ZERO_CHECK\ZERO_CHECK.tlog\".
InitializeBuildStatus:
  Creating "x64\Release\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
CustomBuild:
  Checking Build System
  CMake does not need to re-run because C:/xmr-stak/xmr-stak/build2/CMakeFiles/generate.stamp is up-to-date.
FinalizeBuildStatus:
  Deleting file "x64\Release\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
  Touching "x64\Release\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
Done Building Project "C:\xmr-stak\xmr-stak\build2\ZERO_CHECK.vcxproj" (default targets).

Project "C:\xmr-stak\xmr-stak\build2\install.vcxproj" (1) is building "C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (3) on node 1 (default t
argets).
Project "C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (3) is building "C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj" (4) on node 1 (default
targets).
Project "C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj" (4) is building "C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj" (5) on node 1 (d
efault targets).
Project "C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj" (5) is building "C:\xmr-stak\xmr-stak\build2\xmr-stak-c.vcxproj" (6) on node 1
(default targets).
PrepareForBuild:
  Creating directory "xmr-stak-c.dir\Release\".
  Creating directory "C:\xmr-stak\xmr-stak\build2\bin\Release\".
  Creating directory "xmr-stak-c.dir\Release\xmr-stak-c.tlog\".
InitializeBuildStatus:
  Creating "xmr-stak-c.dir\Release\xmr-stak-c.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
CustomBuild:
  Building Custom Rule C:/xmr-stak/xmr-stak/CMakeLists.txt
  CMake does not need to re-run because C:/xmr-stak/xmr-stak/build2/CMakeFiles/generate.stamp is up-to-date.
ClCompile:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\CL.exe /c /I"C:\xmr-stak\xmr-stak\." /I"C:\Prog
  ram Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include" /I"C:\xmr-stak-dep\libmicrohttpd\include" /I"C:\xmr-stak-dep\openssl\include" /I"C:\xmr-stak-dep\hwloc\i
  nclude" /nologo /W3 /WX- /diagnostics:classic /O2 /Ob2 /D WIN32 /D _WINDOWS /D NDEBUG /D XMR_STAK_LARGEGRID=ON /D XMRSTAK_CUDA_ARCH_LIST=30+35+37+50+52+60+61+62+7
  0 /D GIT_COMMIT_HASH=31a0784 /D GIT_BRANCH=master /D "BACKEND_TYPE=nvidia-amd-cpu" /D _CRT_SECURE_NO_DEPRECATE /D NOMINMAX /D "CMAKE_INTDIR=\"Release\"" /D _MBCS
  /Gm- /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /Fo"xmr-stak-c.dir\Release\\" /Fd"xmr-stak-c.dir\Release\xmr-stak-c.pdb" /Gd /TC /errorReport:queue "
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto\c_blake256.c" "C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto\c_groestl.c" "C:\x
  mr-stak\xmr-stak\xmrstak\backend\cpu\crypto\c_jh.c" "C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto\c_keccak.c" "C:\xmr-stak\xmr-stak\xmrst
  ak\backend\cpu\crypto\c_skein.c"
  c_blake256.c
C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto\c_blake256.c(151): warning C4244: '=': conversion from 'uint64_t' to 'int', possible loss of data [C:\Use
rs\xmr-stak\xmr-stak\build2\xmr-stak-c.vcxproj]
  c_groestl.c
  c_jh.c
  c_keccak.c
  c_skein.c
  Generating Code...
Lib:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\Lib.exe /OUT:"C:\xmr-stak\xmr-stak\build2\bin\R
  elease\xmr-stak-c.lib" /NOLOGO /MACHINE:X64  /machine:x64 "xmr-stak-c.dir\Release\c_blake256.obj"
  "xmr-stak-c.dir\Release\c_groestl.obj"
  "xmr-stak-c.dir\Release\c_jh.obj"
  "xmr-stak-c.dir\Release\c_keccak.obj"
  "xmr-stak-c.dir\Release\c_skein.obj"
  xmr-stak-c.vcxproj -> C:\xmr-stak\xmr-stak\build2\bin\Release\xmr-stak-c.lib
FinalizeBuildStatus:
  Deleting file "xmr-stak-c.dir\Release\xmr-stak-c.tlog\unsuccessfulbuild".
  Touching "xmr-stak-c.dir\Release\xmr-stak-c.tlog\xmr-stak-c.lastbuildstate".
Done Building Project "C:\xmr-stak\xmr-stak\build2\xmr-stak-c.vcxproj" (default targets).

PrepareForBuild:
  Creating directory "xmr-stak-backend.dir\Release\".
  Creating directory "xmr-stak-backend.dir\Release\xmr-stak-backend.tlog\".
InitializeBuildStatus:
  Creating "xmr-stak-backend.dir\Release\xmr-stak-backend.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
CustomBuild:
  Building Custom Rule C:/xmr-stak/xmr-stak/CMakeLists.txt
  CMake does not need to re-run because C:/xmr-stak/xmr-stak/build2/CMakeFiles/generate.stamp is up-to-date.
MakeDirsForCl:
  Creating directory "C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.dir\Release\xmrstak".
  Creating directory "C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.dir\Release\xmrstak\backend\cpu".
ClCompile:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\CL.exe /c /I"C:\xmr-stak\xmr-stak\." /I"C:\Prog
  ram Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include" /I"C:\xmr-stak-dep\libmicrohttpd\include" /I"C:\xmr-stak-dep\openssl\include" /I"C:\xmr-stak-dep\hwloc\i
  nclude" /nologo /W3 /WX- /diagnostics:classic /O2 /Ob2 /D WIN32 /D _WINDOWS /D NDEBUG /D XMR_STAK_LARGEGRID=ON /D XMRSTAK_CUDA_ARCH_LIST=30+35+37+50+52+60+61+62+7
  0 /D GIT_COMMIT_HASH=31a0784 /D GIT_BRANCH=master /D "BACKEND_TYPE=nvidia-amd-cpu" /D _CRT_SECURE_NO_DEPRECATE /D NOMINMAX /D "CMAKE_INTDIR=\"Release\"" /D _MBCS
  /Gm- /EHsc /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /Fo"xmr-stak-backend.dir\Release\/xmrstak/jconf.cpp.obj" /Fd"xmr-stak-backend.dir\Release\x
  mr-stak-backend.pdb" /Gd /TP /errorReport:queue "C:\xmr-stak\xmr-stak\xmrstak\jconf.cpp"
  jconf.cpp
C:\xmr-stak\xmr-stak\xmrstak\jconf.cpp(133): warning C4267: 'argument': conversion from 'size_t' to 'rapidjson::SizeType', possible loss of data [C:\User
s\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\CL.exe /c /I"C:\xmr-stak\xmr-stak\." /I"C:\Prog
  ram Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include" /I"C:\xmr-stak-dep\libmicrohttpd\include" /I"C:\xmr-stak-dep\openssl\include" /I"C:\xmr-stak-dep\hwloc\i
  nclude" /nologo /W3 /WX- /diagnostics:classic /O2 /Ob2 /D WIN32 /D _WINDOWS /D NDEBUG /D XMR_STAK_LARGEGRID=ON /D XMRSTAK_CUDA_ARCH_LIST=30+35+37+50+52+60+61+62+7
  0 /D GIT_COMMIT_HASH=31a0784 /D GIT_BRANCH=master /D "BACKEND_TYPE=nvidia-amd-cpu" /D _CRT_SECURE_NO_DEPRECATE /D NOMINMAX /D "CMAKE_INTDIR=\"Release\"" /D _MBCS
  /Gm- /EHsc /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /Fo"xmr-stak-backend.dir\Release\\" /Fd"xmr-stak-backend.dir\Release\xmr-stak-backend.pdb"
  /Gd /TP /errorReport:queue "C:\xmr-stak\xmr-stak\xmrstak\version.cpp" "C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\hwlocMemory.cpp" "C:\K
  yle\xmr-stak\xmr-stak\xmrstak\backend\cpu\minethd.cpp" "C:\xmr-stak\xmr-stak\xmrstak\backend\backendConnector.cpp" "C:\xmr-stak\xmr-stak\xmr
  stak\backend\globalStates.cpp" "C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto\cryptonight_common.cpp" "C:\xmr-stak\xmr-stak\xmrstak\http\h
  ttpd.cpp" "C:\xmr-stak\xmr-stak\xmrstak\http\webdesign.cpp" "C:\xmr-stak\xmr-stak\xmrstak\misc\console.cpp" "C:\xmr-stak\xmr-stak
  \xmrstak\misc\executor.cpp" "C:\xmr-stak\xmr-stak\xmrstak\misc\telemetry.cpp" "C:\xmr-stak\xmr-stak\xmrstak\misc\utility.cpp" "C:\
  \xmr-stak\xmr-stak\xmrstak\net\jpsock.cpp" "C:\xmr-stak\xmr-stak\xmrstak\net\socket.cpp"
  version.cpp
  hwlocMemory.cpp
C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\hwlocMemory.cpp(37): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of dat
a [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  minethd.cpp
C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of data
[C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\include\xmemory0(945): warning C4267: 'initializing': conversion from 'size_
t' to 'unsigned int', possible loss of data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\include\vector(957): note: see reference to function template instantiatio
  n 'void std::_Default_allocator_traits<_Alloc>::construct<_Ty,size_t&>(_Alloc &,_Objty *const ,size_t &)' being compiled
          with
          [
              _Alloc=std::allocator<std::_Vbase>,
              _Ty=unsigned int,
              _Objty=unsigned int
          ]
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\include\vector(957): note: see reference to function template instantiatio
  n 'void std::_Default_allocator_traits<_Alloc>::construct<_Ty,size_t&>(_Alloc &,_Objty *const ,size_t &)' being compiled
          with
          [
              _Alloc=std::allocator<std::_Vbase>,
              _Ty=unsigned int,
              _Objty=unsigned int
          ]
  c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\autoAdjustHwloc.hpp(210): note: see reference to function template instantiation 'void std::vector<std::_Vbase
  ,std::allocator<std::_Vbase>>::emplace_back<size_t&>(size_t &)' being compiled
c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(293): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of
data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\minethd.cpp(359): note: see reference to function template instantiation 'void cryptonight_hash<2097136,524288
  ,2097152,false,false>(const void *,::size_t,void *,cryptonight_ctx *)' being compiled
c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(358): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of
data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\minethd.cpp(508): note: see reference to function template instantiation 'void cryptonight_double_hash<2097136
  ,524288,2097152,false,false>(const void *,::size_t,void *,cryptonight_ctx **)' being compiled
c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(359): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of
data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(492): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of
data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\minethd.cpp(508): note: see reference to function template instantiation 'void cryptonight_triple_hash<2097136
  ,524288,2097152,false,false>(const void *,::size_t,void *,cryptonight_ctx **)' being compiled
c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(567): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of
data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\minethd.cpp(508): note: see reference to function template instantiation 'void cryptonight_quad_hash<2097136,5
  24288,2097152,false,false>(const void *,::size_t,void *,cryptonight_ctx **)' being compiled
c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(655): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of
data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\minethd.cpp(508): note: see reference to function template instantiation 'void cryptonight_penta_hash<2097136,
  524288,2097152,false,false>(const void *,::size_t,void *,cryptonight_ctx **)' being compiled
C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\minethd.cpp(639): warning C4267: 'argument': conversion from 'size_t' to 'uint32_t', possible loss of data [C:\U
sers\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\minethd.cpp(528): note: see reference to function template instantiation 'void xmrstak::cpu::minethd::multiway
  _work_main<2>(xmrstak::cpu::minethd::cn_hash_fun_multi)' being compiled
  backendConnector.cpp
  globalStates.cpp
  cryptonight_common.cpp
  httpd.cpp
C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of data
[C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  webdesign.cpp
  console.cpp
C:\xmr-stak\xmr-stak\xmrstak\misc\console.cpp(232): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data [C:\xm
r-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  executor.cpp
c:\xmr-stak\xmr-stak\xmrstak\misc\executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of data [C
:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  telemetry.cpp
C:\xmr-stak\xmr-stak\xmrstak\misc\telemetry.cpp(93): warning C4244: '=': conversion from 'uint64_t' to 'double', possible loss of data [C:\xmr
-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\misc\telemetry.cpp(94): warning C4244: '=': conversion from 'uint64_t' to 'double', possible loss of data [C:\xmr
-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  utility.cpp
  jpsock.cpp
c:\xmr-stak\xmr-stak\xmrstak\net\socks.hpp(38): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:\xmr-
stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
c:\xmr-stak\xmr-stak\xmrstak\net\socks.hpp(51): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:\xmr-
stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of data
[C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\net\jpsock.cpp(232): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\net\jpsock.cpp(250): warning C4244: 'initializing': conversion from '__int64' to 'int', possible loss of data [C:\
\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\net\jpsock.cpp(567): warning C4267: 'argument': conversion from 'size_t' to 'rapidjson::SizeType', possible loss of data [C:
\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  socket.cpp
c:\xmr-stak\xmr-stak\xmrstak\net\socks.hpp(38): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:\xmr-
stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
c:\xmr-stak\xmr-stak\xmrstak\net\socks.hpp(51): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:\xmr-
stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of data
[C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\net\socket.cpp(150): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data [C:\
xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  Generating Code...
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\CL.exe /c /I"C:\xmr-stak\xmr-stak\." /I"C:\Prog
  ram Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include" /I"C:\xmr-stak-dep\libmicrohttpd\include" /I"C:\xmr-stak-dep\openssl\include" /I"C:\xmr-stak-dep\hwloc\i
  nclude" /nologo /W3 /WX- /diagnostics:classic /O2 /Ob2 /D WIN32 /D _WINDOWS /D NDEBUG /D XMR_STAK_LARGEGRID=ON /D XMRSTAK_CUDA_ARCH_LIST=30+35+37+50+52+60+61+62+7
  0 /D GIT_COMMIT_HASH=31a0784 /D GIT_BRANCH=master /D "BACKEND_TYPE=nvidia-amd-cpu" /D _CRT_SECURE_NO_DEPRECATE /D NOMINMAX /D "CMAKE_INTDIR=\"Release\"" /D _MBCS
  /Gm- /EHsc /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /Fo"xmr-stak-backend.dir\Release\/xmrstak/backend/cpu/jconf.cpp.obj" /Fd"xmr-stak-backend.d
  ir\Release\xmr-stak-backend.pdb" /Gd /TP /errorReport:queue "C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\jconf.cpp"
  jconf.cpp
C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\jconf.cpp(106): warning C4267: 'argument': conversion from 'size_t' to 'rapidjson::SizeType', possible loss of d
ata [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
Lib:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\Lib.exe /OUT:"C:\xmr-stak\xmr-stak\build2\bin\R
  elease\xmr-stak-backend.lib" /NOLOGO /MACHINE:X64  /machine:x64 "xmr-stak-backend.dir\Release\version.obj"
  "xmr-stak-backend.dir\Release\hwlocMemory.obj"
  "xmr-stak-backend.dir\Release\minethd.obj"
  "xmr-stak-backend.dir\Release\backendConnector.obj"
  "xmr-stak-backend.dir\Release\globalStates.obj"
  "xmr-stak-backend.dir\Release\cryptonight_common.obj"
  "xmr-stak-backend.dir\Release\httpd.obj"
  "xmr-stak-backend.dir\Release\webdesign.obj"
  "xmr-stak-backend.dir\Release\console.obj"
  "xmr-stak-backend.dir\Release\executor.obj"
  "xmr-stak-backend.dir\Release\telemetry.obj"
  "xmr-stak-backend.dir\Release\utility.obj"
  "xmr-stak-backend.dir\Release\jpsock.obj"
  "xmr-stak-backend.dir\Release\socket.obj"
  "xmr-stak-backend.dir\Release\/xmrstak/jconf.cpp.obj"
  "xmr-stak-backend.dir\Release\/xmrstak/backend/cpu/jconf.cpp.obj"
  xmr-stak-backend.vcxproj -> C:\xmr-stak\xmr-stak\build2\bin\Release\xmr-stak-backend.lib
FinalizeBuildStatus:
  Deleting file "xmr-stak-backend.dir\Release\xmr-stak-backend.tlog\unsuccessfulbuild".
  Touching "xmr-stak-backend.dir\Release\xmr-stak-backend.tlog\xmr-stak-backend.lastbuildstate".
Done Building Project "C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj" (default targets).

PrepareForBuild:
  Creating directory "xmr-stak.dir\Release\".
  Creating directory "xmr-stak.dir\Release\xmr-stak.tlog\".
InitializeBuildStatus:
  Creating "xmr-stak.dir\Release\xmr-stak.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
CustomBuild:
  Building Custom Rule C:/xmr-stak/xmr-stak/CMakeLists.txt
  CMake does not need to re-run because C:/xmr-stak/xmr-stak/build2/CMakeFiles/generate.stamp is up-to-date.
ClCompile:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\CL.exe /c /I"C:\xmr-stak\xmr-stak\." /I"C:\Prog
  ram Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include" /I"C:\xmr-stak-dep\libmicrohttpd\include" /I"C:\xmr-stak-dep\openssl\include" /I"C:\xmr-stak-dep\hwloc\i
  nclude" /nologo /W3 /WX- /diagnostics:classic /O2 /Ob2 /D WIN32 /D _WINDOWS /D NDEBUG /D XMR_STAK_LARGEGRID=ON /D XMRSTAK_CUDA_ARCH_LIST=30+35+37+50+52+60+61+62+7
  0 /D GIT_COMMIT_HASH=31a0784 /D GIT_BRANCH=master /D "BACKEND_TYPE=nvidia-amd-cpu" /D _CRT_SECURE_NO_DEPRECATE /D NOMINMAX /D "CMAKE_INTDIR=\"Release\"" /D _MBCS
  /Gm- /EHsc /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /Fo"xmr-stak.dir\Release\\" /Fd"xmr-stak.dir\Release\vc141.pdb" /Gd /TP /errorReport:queue
  "C:\xmr-stak\xmr-stak\xmrstak\cli\cli-miner.cpp"
  cli-miner.cpp
C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of data
[C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\cli\cli-miner.cpp(357): warning C4244: 'argument': conversion from 'time_t' to 'unsigned int', possible loss of data [C:\Use
rs\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\cli\cli-miner.cpp(678): warning C4244: 'initializing': conversion from 'unsigned __int64' to 'double', possible loss of data
 [C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj]
Link:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE /OUT:"C:\xmr-stak\x
  mr-stak\build2\bin\Release\xmr-stak.exe" /INCREMENTAL:NO /NOLOGO /LIBPATH:"C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.1/lib/x64/OpenCL.lib" /LIBPATH:"C
  :/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.1/lib/x64/OpenCL.lib/Release" "C:\xmr-stak-dep\libmicrohttpd\lib\libmicrohttpd.lib" "C:\xmr-stak-dep\openssl\
  lib\ssleay32.lib" "C:\xmr-stak-dep\openssl\lib\libeay32.lib" "C:\xmr-stak-dep\hwloc\lib\libhwloc.lib" wsock32.lib ws2_32.lib "bin\Release\xmr-stak-c.lib" "bin\Rel
  ease\xmr-stak-backend.lib" "bin\Release\xmr-stak-c.lib" "C:\xmr-stak-dep\libmicrohttpd\lib\libmicrohttpd.lib" "C:\xmr-stak-dep\openssl\lib\ssleay32.lib" "C:\xmr-s
  tak-dep\openssl\lib\libeay32.lib" "C:\xmr-stak-dep\hwloc\lib\libhwloc.lib" wsock32.lib ws2_32.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32
  .lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /PDB:"C:/xmr-stak/xmr-
  stak/build2/bin/Release/xmr-stak.pdb" /SUBSYSTEM:CONSOLE /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"C:/xmr-stak/xmr-stak/build2/bin/Release/xmr-stak.lib"
   /MACHINE:X64  /machine:x64 "xmr-stak.dir\Release\cli-miner.obj"
  libmicrohttpd.lib(daemon.obj) : MSIL .netmodule or module compiled with /GL found; restarting link with /LTCG; add /LTCG to the link command line to improve linke
  r performance
  Generating code
  Finished generating code
  xmr-stak.vcxproj -> C:\xmr-stak\xmr-stak\build2\bin\Release\xmr-stak.exe
FinalizeBuildStatus:
  Deleting file "xmr-stak.dir\Release\xmr-stak.tlog\unsuccessfulbuild".
  Touching "xmr-stak.dir\Release\xmr-stak.tlog\xmr-stak.lastbuildstate".
Done Building Project "C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj" (default targets).

Project "C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (3) is building "C:\xmr-stak\xmr-stak\build2\xmrstak_cuda_backend.vcxproj" (7) on node
 1 (default targets).
PrepareForBuild:
  Creating directory "xmrstak_cuda_backend.dir\Release\".
  Creating directory "xmrstak_cuda_backend.dir\Release\xmrstak_.98454FD2.tlog\".
InitializeBuildStatus:
  Creating "xmrstak_cuda_backend.dir\Release\xmrstak_.98454FD2.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
ComputeCustomBuildOutput:
  Creating directory "C:\xmr-stak\xmr-stak\build2\CMakeFiles\xmrstak_cuda_backend.dir\xmrstak\backend\nvidia\nvcc_code\Release\".
CustomBuild:
  Building NVCC (Device) object CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/Release/xmrstak_cuda_backend_generated_cuda_core.cu.obj
  CMake Error at xmrstak_cuda_backend_generated_cuda_core.cu.obj.Release.cmake:219 (message):
    Error generating
    C:/xmr-stak/xmr-stak/build2/CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/Release/xmrstak_cuda_backend_generated_cuda_core.cu.
  obj


C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe" exited with code
 1. [C:\xmr-stak\xmr-stak\build2\xmrstak_cuda_backend.vcxproj]
Done Building Project "C:\xmr-stak\xmr-stak\build2\xmrstak_cuda_backend.vcxproj" (default targets) -- FAILED.

Project "C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (3) is building "C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj" (8) on no
de 1 (default targets).
PrepareForBuild:
  Creating directory "xmrstak_opencl_backend.dir\Release\".
  Creating directory "xmrstak_opencl_backend.dir\Release\xmrstak_.3AFD030C.tlog\".
InitializeBuildStatus:
  Creating "xmrstak_opencl_backend.dir\Release\xmrstak_.3AFD030C.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
CustomBuild:
  Building Custom Rule C:/xmr-stak/xmr-stak/CMakeLists.txt
  CMake does not need to re-run because C:/xmr-stak/xmr-stak/build2/CMakeFiles/generate.stamp is up-to-date.
ClCompile:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\CL.exe /c /I"C:\xmr-stak\xmr-stak\." /I"C:\Prog
  ram Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include" /I"C:\xmr-stak-dep\libmicrohttpd\include" /I"C:\xmr-stak-dep\openssl\include" /I"C:\xmr-stak-dep\hwloc\i
  nclude" /nologo /W3 /WX- /diagnostics:classic /O2 /Ob2 /D WIN32 /D _WINDOWS /D NDEBUG /D XMR_STAK_LARGEGRID=ON /D XMRSTAK_CUDA_ARCH_LIST=30+35+37+50+52+60+61+62+7
  0 /D GIT_COMMIT_HASH=31a0784 /D GIT_BRANCH=master /D "BACKEND_TYPE=nvidia-amd-cpu" /D _CRT_SECURE_NO_DEPRECATE /D NOMINMAX /D "CMAKE_INTDIR=\"Release\"" /D xmrsta
  k_opencl_backend_EXPORTS /D _WINDLL /D _MBCS /Gm- /EHsc /MD /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /Fo"xmrstak_opencl_backend.dir\Release\\" /Fd"
  xmrstak_opencl_backend.dir\Release\vc141.pdb" /Gd /TP /errorReport:queue "C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp" "C:\xmr-
  stak\xmr-stak\xmrstak\backend\amd\jconf.cpp" "C:\xmr-stak\xmr-stak\xmrstak\backend\amd\minethd.cpp"
  gpu.cpp
C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(33): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:\U
sers\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(555): warning C4018: '<': signed/unsigned mismatch [C:\xmr-stak\xmr-stak\build2\xmrst
ak_opencl_backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(662): warning C4267: 'argument': conversion from 'size_t' to 'cl_uint', possible loss of data [C
:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(699): warning C4267: '=': conversion from 'size_t' to 'cl_int', possible loss of data [C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(981): warning C4267: '+=': conversion from 'size_t' to 'uint32_t', possible loss of data [C:\Use
rs\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  jconf.cpp
C:\xmr-stak\xmr-stak\xmrstak\backend\amd\jconf.cpp(101): warning C4267: 'argument': conversion from 'size_t' to 'rapidjson::SizeType', possible loss of d
ata [C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  minethd.cpp
c:\xmr-stak\xmr-stak\xmrstak\backend\amd\autoAdjust.hpp(60): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data [C:\
xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of data
[C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
C:\xmr-stak\xmr-stak\xmrstak\backend\amd\minethd.cpp(211): warning C4267: 'initializing': conversion from 'size_t' to 'uint32_t', possible loss of data [
C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  Generating Code...
Link:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE /OUT:"C:\xmr-stak\x
  mr-stak\build2\bin\Release\xmrstak_opencl_backend.dll" /INCREMENTAL:NO /NOLOGO /LIBPATH:"C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.1/lib/x64/OpenCL.li
  b" /LIBPATH:"C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.1/lib/x64/OpenCL.lib/Release" "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\lib\x64\
  OpenCL.lib" "bin\Release\xmr-stak-backend.lib" "bin\Release\xmr-stak-c.lib" "C:\xmr-stak-dep\libmicrohttpd\lib\libmicrohttpd.lib" "C:\xmr-stak-dep\openssl\lib\ssl
  eay32.lib" "C:\xmr-stak-dep\openssl\lib\libeay32.lib" "C:\xmr-stak-dep\hwloc\lib\libhwloc.lib" wsock32.lib ws2_32.lib kernel32.lib user32.lib gdi32.lib winspool.l
  ib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /PDB:"C:/xmr-stak/xmr-stak/build2/bin/Release/xmrstak_opencl_backend.pdb" /SUBSYSTEM:CONSOLE /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"C:/xmr-stak/xmr-sta
  k/build2/bin/Release/xmrstak_opencl_backend.lib" /MACHINE:X64  /machine:x64 /DLL xmrstak_opencl_backend.dir\Release\gpu.obj
  xmrstak_opencl_backend.dir\Release\jconf.obj
  xmrstak_opencl_backend.dir\Release\minethd.obj
  libhwloc.lib(topology.obj) : MSIL .netmodule or module compiled with /GL found; restarting link with /LTCG; add /LTCG to the link command line to improve linker p
  erformance
     Creating library C:/xmr-stak/xmr-stak/build2/bin/Release/xmrstak_opencl_backend.lib and object C:/xmr-stak/xmr-stak/build2/bin/Release/xm
  rstak_opencl_backend.exp
  Generating code
  Finished generating code
  xmrstak_opencl_backend.vcxproj -> C:\xmr-stak\xmr-stak\build2\bin\Release\xmrstak_opencl_backend.dll
FinalizeBuildStatus:
  Deleting file "xmrstak_opencl_backend.dir\Release\xmrstak_.3AFD030C.tlog\unsuccessfulbuild".
  Touching "xmrstak_opencl_backend.dir\Release\xmrstak_.3AFD030C.tlog\xmrstak_opencl_backend.lastbuildstate".
Done Building Project "C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj" (default targets).

Done Building Project "C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (default targets) -- FAILED.

Done Building Project "C:\xmr-stak\xmr-stak\build2\install.vcxproj" (default targets) -- FAILED.


Build FAILED.

"C:\xmr-stak\xmr-stak\build2\install.vcxproj" (default target) (1) ->
"C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (default target) (3) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj" (default target) (4) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj" (default target) (5) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak-c.vcxproj" (default target) (6) ->
(ClCompile target) ->
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto\c_blake256.c(151): warning C4244: '=': conversion from 'uint64_t' to 'int', possible loss of data [C:\U
sers\xmr-stak\xmr-stak\build2\xmr-stak-c.vcxproj]


"C:\xmr-stak\xmr-stak\build2\install.vcxproj" (default target) (1) ->
"C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (default target) (3) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj" (default target) (4) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj" (default target) (5) ->
  C:\xmr-stak\xmr-stak\xmrstak\jconf.cpp(133): warning C4267: 'argument': conversion from 'size_t' to 'rapidjson::SizeType', possible loss of data [C:\Us
ers\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]


"C:\xmr-stak\xmr-stak\build2\install.vcxproj" (default target) (1) ->
"C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (default target) (3) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj" (default target) (4) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj" (default target) (5) ->
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\hwlocMemory.cpp(37): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of d
ata [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of dat
a [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\include\xmemory0(945): warning C4267: 'initializing': conversion from 'siz
e_t' to 'unsigned int', possible loss of data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(293): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss o
f data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(358): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss o
f data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(359): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss o
f data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(492): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss o
f data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(567): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss o
f data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\backend\cpu\crypto/cryptonight_aesni.h(655): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss o
f data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\minethd.cpp(639): warning C4267: 'argument': conversion from 'size_t' to 'uint32_t', possible loss of data [C:
\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of dat
a [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\misc\console.cpp(232): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data [C:\
xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\misc\executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of data
[C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\misc\telemetry.cpp(93): warning C4244: '=': conversion from 'uint64_t' to 'double', possible loss of data [C:\x
mr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\misc\telemetry.cpp(94): warning C4244: '=': conversion from 'uint64_t' to 'double', possible loss of data [C:\x
mr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\net\socks.hpp(38): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:\xm
r-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\net\socks.hpp(51): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:\xm
r-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of dat
a [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\net\jpsock.cpp(232): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of data [C:\User
s\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\net\jpsock.cpp(250): warning C4244: 'initializing': conversion from '__int64' to 'int', possible loss of data [C:\Ky
le\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\net\jpsock.cpp(567): warning C4267: 'argument': conversion from 'size_t' to 'rapidjson::SizeType', possible loss of data [
C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\net\socks.hpp(38): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:\xm
r-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\net\socks.hpp(51): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:\xm
r-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of dat
a [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\net\socket.cpp(150): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data [C:\Kyl
e\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]


"C:\xmr-stak\xmr-stak\build2\install.vcxproj" (default target) (1) ->
"C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (default target) (3) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj" (default target) (4) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj" (default target) (5) ->
  C:\xmr-stak\xmr-stak\xmrstak\backend\cpu\jconf.cpp(106): warning C4267: 'argument': conversion from 'size_t' to 'rapidjson::SizeType', possible loss of
 data [C:\xmr-stak\xmr-stak\build2\xmr-stak-backend.vcxproj]


"C:\xmr-stak\xmr-stak\build2\install.vcxproj" (default target) (1) ->
"C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (default target) (3) ->
"C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj" (default target) (4) ->
  C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of dat
a [C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\cli\cli-miner.cpp(357): warning C4244: 'argument': conversion from 'time_t' to 'unsigned int', possible loss of data [C:\U
sers\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\cli\cli-miner.cpp(678): warning C4244: 'initializing': conversion from 'unsigned __int64' to 'double', possible loss of da
ta [C:\xmr-stak\xmr-stak\build2\xmr-stak.vcxproj]


"C:\xmr-stak\xmr-stak\build2\install.vcxproj" (default target) (1) ->
"C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (default target) (3) ->
"C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj" (default target) (8) ->
  C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(33): warning C4267: 'argument': conversion from 'size_t' to 'DWORD', possible loss of data [C:
\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(555): warning C4018: '<': signed/unsigned mismatch [C:\xmr-stak\xmr-stak\build2\xmr
stak_opencl_backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(662): warning C4267: 'argument': conversion from 'size_t' to 'cl_uint', possible loss of data
[C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(699): warning C4267: '=': conversion from 'size_t' to 'cl_int', possible loss of data [C:\User
s\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\amd\amd_gpu\gpu.cpp(981): warning C4267: '+=': conversion from 'size_t' to 'uint32_t', possible loss of data [C:\U
sers\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\amd\jconf.cpp(101): warning C4267: 'argument': conversion from 'size_t' to 'rapidjson::SizeType', possible loss of
 data [C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  c:\xmr-stak\xmr-stak\xmrstak\backend\amd\autoAdjust.hpp(60): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data [C
:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  C:\xmr-stak\xmr-stak\.\xmrstak/misc/executor.hpp(67): warning C4244: 'initializing': conversion from 'double' to 'const ::size_t', possible loss of dat
a [C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]
  C:\xmr-stak\xmr-stak\xmrstak\backend\amd\minethd.cpp(211): warning C4267: 'initializing': conversion from 'size_t' to 'uint32_t', possible loss of data
 [C:\xmr-stak\xmr-stak\build2\xmrstak_opencl_backend.vcxproj]


"C:\xmr-stak\xmr-stak\build2\install.vcxproj" (default target) (1) ->
"C:\xmr-stak\xmr-stak\build2\ALL_BUILD.vcxproj" (default target) (3) ->
"C:\xmr-stak\xmr-stak\build2\xmrstak_cuda_backend.vcxproj" (default target) (7) ->
(CustomBuild target) ->
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(171,5): error MSB6006: "cmd.exe" exited with co
de 1. [C:\xmr-stak\xmr-stak\build2\xmrstak_cuda_backend.vcxproj]

    40 Warning(s)
    1 Error(s)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -191,7 +191,9 @@ if(CUDA_ENABLE)
                 set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}" "-D_MWAITXINTRIN_H_INCLUDED")
             endif()
 
-            if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC" AND CUDA_VERSION VERSION_EQUAL 9.0)
+            if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC" AND
+                (CUDA_VERSION VERSION_EQUAL 9.0 OR CUDA_VERSION VERSION_EQUAL 9.1)
+            )
                 # workaround find_package(CUDA) is using the wrong path to the CXX host compiler
                 # overwrite the CUDA host compiler variable with the used CXX MSVC
                 set(CUDA_HOST_COMPILER ${CMAKE_CXX_COMPILER} CACHE FILEPATH "Host side compiler used by NVCC" FORCE)

